tests/common-pg.cpp

```diff
@@ -129,7 +129,7 @@ const std::string &tempdb::conninfo() const {
     return m_conninfo;
 }
 
-void tempdb::setup_extension(conn_ptr db, const std::string &extension, ...) {
+void tempdb::setup_extension(conn_ptr db, const std::string extension, ...) {
```

Comment:
The whole function should be refactored, probably taking a `std::vector<std::string>&` as argument instead of the variable argument list. This would make the code much cleaner and you don't have problems with references vs. values.